### ISSUE TITLE
[CAIA Intake] Disability Benefits: Contention Classification Team - New Conditions Page (Form: 21-526EZ)

### ISSUE THREAD

**Issue by sudeeptibh** [department-of-veterans-affairs/va.gov-team#75567]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Contention Classification 
- **OCTO product owner:** `Jen Bertsch`
- **Product name:** `526EZ`
- **Product manager:**  `Yurena Garcia`
- **Slack channel:** #[benefits-contention-classification ](https://join.slack.com/share/enQtNjYwMDExOTQyMDc1Ny1iOGNlYTE4ZjY4OGM2YmUzOWE3YmY3MTMwMzA4MzRkMzQyNWVjMzE3NDQ0ODk4NDMzYmJhNjNmNGRiN2FjM2Vj)
- **Dedicated content writer on your team (if you have one):** `N/A`
- **Dedicated a11y specialist on your team (if you have one):** `N/A`
 
## About your initiative
We are updating the "add new conditions" page on the 526EZ Form. We started with updating the conditions themselves, the list that the autosuggestion component (dropdown) uses. Now, we are focused on UI and copy changes.  

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

`Tell us briefly about your work and the outcomes you’re aiming to achieve.`
Our goal with the new conditions page is to improve user experience and accuracy when entering medical conditions for disability benefits. We aim to simplify the process by creating a user-friendly interface that enables Veterans to select or input their conditions effortlessly. Specifically, we aim to ensure that users understand how the autosuggest component functions, what information they need to provide, and that it supports free-text entries.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
`Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.`

We are in the wireframing phase. We want a prototype to be built and ready to test by March 13th. 

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.
- [] We have not started the collaboration cycle yet and will begin the Design intent in our next Sprint. 

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Mural board for wireframes](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1689885142486/f54c64b6eb52b4ac7eca4fb926a946aa072f2ed8?wid=0-1706739182333 )

There are two variations: "removing additional information" and "Adding additional information to component". We can test variations of both options or consolidate to one if there's a strong opinion. 

[Codepen for accessibility testing 
](https://codepen.io/ccteam/live/ZEZeMPR)

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
- [x] Wireframes Shared
- [x] Building CodePen prototype (goal March 13) share with CAIA (a11ys to conduct audit)
- [x] Provide CAIA a single content source of truth document (per Kickoff Meeting 2/21/2024)
```

```[tasklist]
### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl
- [x] Kickoff meeting scheduled for 2/21 - 12:30 PM ET
- [ ] Review next steps progress from 2/21 meeting
```

```[tasklist]
### CAIA a11y tasks
- [x] Participate in Pre-Kickoff 2/21/2024 (Sara/Eli/Jamie/Terry)
- [x] Participate in Kickoff 2/21/2024
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76707
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76779
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76766
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76919
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76919
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/84006
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/84007
```

**Comment by sudeeptibh** [comment]
Source of truth document - https://dvagov.sharepoint.com/:w:/r/sites/vaabdvro/_layouts/15/Doc.aspx?sourcedoc=%7BFEED00F6-8049-4FBF-8669-74CEECB16C8D%7D&file=Improved%20autosuggest%20component%20-%20Content%20for%20CAIA%20Review.docx&action=default&mobileredirect=true 

I added VA emails for the folks on this ticket, let me know if anyone else needs access. Thanks!

**Comment by aprocik1** [comment]
Hey @sudeeptibh, can you add Anthony.Procik@va.gov and Katherine.Fung@va.gov as editors to your content source of truth? Thanks!

**Comment by aprocik1** [comment]
Hi @sudeeptibh, Katherine and I left our content recommendations in your [source of truth SharePoint document](https://dvagov.sharepoint.com/:w:/s/vaabdvro/EfYA7f5JgL9Phml0zuyxbI0BQSWo8iRrbYzTGJzmMe_-JA?e=ekdirl). Let me know if you have any questions. I'm happy to return to this!

**Comment by sudeeptibh** [comment]
@aprocik1 Thank you so much!! Would it be possible to get on a call and ask clarifying questions? I can put some time on your calendar.

**Comment by aprocik1** [comment]
@sudeeptibh, feel free to schedule something for us! My Google calendar is up-to-date.

**Comment by coforma-jamie** [comment]
@sudeeptibh, I've completed the accessibility review. I tagged you on [Slack](https://dsva.slack.com/archives/C04AZ8T7XN1/p1711464278139999?thread_ts=1711147278.486599&cid=C04AZ8T7XN1) as well, here's the [audit spreadsheet](https://docs.google.com/spreadsheets/d/1jV7i25O3xHoT1rJghUn-WTo3AiLygxbGXMuD4EJjdjY/edit#gid=0). Please let me know if you have any questions!

**Comment by sudeeptibh** [comment]
@aprocik1 we are in the process of evaluating all the error messages on the page. Would you be able to do a review of the error message copy sometime in the next couple of weeks? 

The source of truth content for error messages is in our [test cases spreadsheet ](https://docs.google.com/spreadsheets/d/12DaRQF6iwm_rUOhmgMht0h00HTJPoIa0YMxVCmHhzLU/edit#gid=1618556630)in the "error messages" tab. There's a column for CAIA review.

**Comment by aprocik1** [comment]
Thanks for submitting this intake, @sudeeptibh. We have a handful of projects in the queue to close out before we can turn to this. By when would you like us to complete this review?

**Comment by sudeeptibh** [comment]
No worries, we can wait our turn! Is 2 weeks reasonable? @aprocik1  It's for 8 1-2 sentence error messages.

**Comment by sudeeptibh** [comment]
@coforma-jamie, we aim to be done building in staging by May 27th and launching around June 10th. Would it be possible to book your time for accessibility testing in staging pre-launch between May 27 - June 10th?

**Comment by coforma-jamie** [comment]
@sudeeptibh Yes! That works for me. Keep me posted when you're ready for an audit, and send over the staging URL and any test user info.

**Comment by coforma-jamie** [comment]
Question @sudeeptibh - are you still planning to do any user research with assistive tech users before (or after?) launch? 
#76779 
#76707

**Comment by sudeeptibh** [comment]
> Question @sudeeptibh - are you still planning to do any user research with assistive tech users before (or after?) launch? #76779 #76707

Yes! We are planning on doing it after launch

**Comment by aprocik1** [comment]
@sudeeptibh, I reviewed your error messages and drafted some recommendations. Before we finalize those recommendations, I left 3 questions for you in cells G2, G4, and G5.

**Comment by sudeeptibh** [comment]
update - we will kick off recruiting next week with the goal of a 2-week recruitment period and to do the usability research week of July 22. https://github.com/department-of-veterans-affairs/va.gov-team/issues/76779

**Comment by aprocik1** [comment]
Hi @sudeeptibh! I think our work on this ticket is complete. Can I close it?

**Comment by sudeeptibh** [comment]
Yes!

*Sudeepti Bhatnagar (she/her)*
Senior Designer / Researcher
734-255-8989
navapbc.com
